ezGroceriesShoppingList/src/main/java/com/ezgroceries/shoppinglist/entity/ShoppingList.java

```diff
@@ -0,0 +1,48 @@
+package com.ezgroceries.shoppinglist.entity;
+
+public class ShoppingList {
+
+    private String shoppingListId;
+    private String name;
+
+    public ShoppingList(String shoppingListId, String name) {
+        this.shoppingListId = shoppingListId;
+        this.name = name;
+    }
+
+    private String[] ingredients;
```

Comment:
For future references, if you join a team where there is no lombok and you must provide getters setter etc.. you should follow a structure
 -> first part of the class will be all the fields you provide 
 -> second part should be constructors 
 -> third part will be the getter setters and other functions related to your class